test/e2e/config/azure-dev.yaml

```diff
@@ -154,6 +154,8 @@ variables:
   AKS_KUBERNETES_VERSION: "latest"
   AKS_KUBERNETES_VERSION_UPGRADE_FROM: "latest-1"
   KUBERNETES_VERSION: "${KUBERNETES_VERSION:-stable-1.24}"
+  FLATCAR_KUBERNETES_VERSION: "${FLATCAR_KUBERNETES_VERSION:-stable-1.24}"
```

Comment:
Could this default to `${KUBERNETES_VERSION}` or does `${FLATCAR_KUBERNETES_VERSION}` need to be updated separately?

Reply:
It could default to it though, just may need to be adjusted in case we lag with the image as for Flatcar. Whatever you think it's better.